bot/cogs/memes_cog/memes_cog.py

```diff
@@ -104,22 +129,23 @@ async def spongebob(self, ctx, *, args):
         await ctx.send(embed=embed)
 
     @commands.command(aliases=['rave', '🦀'])
-    async def crab(self, ctx, *, args='Bottom text\n is dead'):
+    async def crab(self, ctx, is_rave: t.Optional[bool] = True, *, args='Bottom text\n is dead'):
         '''
         Create your own crab rave.
-
-        Usage: <prefix>crab [text=Bottom text\n is dead]
+        Usage: <prefix>crab [is_rave=True] [text=Bottom text\\n is dead]
         Aliases: rave, 🦀
         '''
         # crab.gif dimensions - 352 by 200
         # Immediately grab the timestamp incase of multiple calls in a row
         timestamp = datetime.datetime.utcnow()
         msg = await ctx.send('Generating your gif')
-        # Open crab.gif and add our font
-        im = Image.open('bot/cogs/memes_cog/assets/crab.gif')
-        fnt_path = 'bot/cogs/memes_cog/assets/LemonMilk.otf'
-        fnt = ImageFont.truetype(fnt_path, 11)
 
+        # Determine if there is a full rave or not
+        if is_rave:
```

Comment:
we dont need this, you can just use is_rave directly and cast the bool to an int in the pillow_process function